src/main/java/org/broadinstitute/dsde/consent/ontology/translate/utils/LoadUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsde.consent.ontology.translate.utils;
+
+import com.github.jsonldjava.shaded.com.google.common.reflect.TypeToken;
+import com.google.gson.Gson;
+import org.broadinstitute.dsde.consent.ontology.translate.model.TermItem;
+import org.parboiled.common.FileUtils;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+public class LoadUtils {
+    public static List<TermItem> termsFromLocalResourcesJSON(@NotNull String jsonFilePath) throws Exception {
```

Comment:
I would leave this inside `Translate` until you have another use for it outside of that class. In general unless you know something will be useful as a separate "utility" method I don't think it's worth making a separate class for it.